src/harness/sas_testcase.py

```diff
@@ -29,4 +29,31 @@ def AssertContainsRequiredFields(self, schema_filename, response):
     schema_dir = os.path.dirname(os.path.realpath(schema_filename))
     resolver = RefResolver(referrer=schema, base_uri='file://' + schema_dir + '/')
     # Raises ValidationError when incorrect response
-    validate(response, schema, resolver=resolver)
\ No newline at end of file
+    validate(response, schema, resolver=resolver)
+
+  def AssertValidResponseFormatForApprovedGrant(self, grant_response):
```

Comment:
FYI I think Python naming convention says this should start with a lowercase character: https://www.python.org/dev/peps/pep-0008/#function-names